R/endpoints.R

```diff
@@ -364,6 +361,21 @@ api_root <- function() {
   scalar("Welcome to hintr")
 }
 
+prepare_status_response <- function(value, id) {
+  set_scalar <- function(x) {
+    if (length(names(x)) > 1) {
+      lapply(x, set_scalar)
+    } else {
+      scalar(x)
+    }
+  }
+
+  response_value <- lapply(value[-which(names(value) == "progress")], scalar)
```

Comment:
```
lapply(value[names(value) != "progress"], scalar)
```
perhaps?

Reply:
Much nicer thank you. I thought what I wrote was painfully complicated for what I was trying to achieve. I think I just had a confused moment and forgot `!=` existed.